src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -1032,4 +1032,7 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Text_PackageMappingsAutoCreate" xml:space="preserve">
     <value>A package source mapping will be created.</value>
   </data>
+  <data name="Checkbox_Show_Vulnerable_Only" xml:space="preserve">
+    <value>Show vulnerable only</value>
```

Comment:
Have we validated this with @JonDouglas or @albarry4?

Reply:
Yep, this is fine as in IMO. Here's some alternatives if needed.

"Show vulnerable"

"Show vulnerable packages"

etc.

Reply:
"Show only vulnerable" might read better